run_tests.sh

```diff
@@ -17,7 +17,7 @@ set -ex
 go version
 
 # run tests on all modules
-ROOTPATH=$(go list -m)
+ROOTPATH=$(go list)
```

Comment:
FWIW, `ROOTPATH` is really a misnomer: it should be `ROOTMOD` or `ROOTPKG`, because at this point in the script we're not interested in the path, but rather in the root module name (to remove as a prefix in the loop).

Reply:
I made this change in #3022.